common/src/main/scala/com/snowplowanalytics/snowplow/storage/bigquery/common/ProcessorInfo.scala

```diff
@@ -0,0 +1,25 @@
+package com.snowplowanalytics.snowplow.storage.bigquery.common
+
+import io.circe.{Encoder, Json}
+import io.circe.syntax._
+import com.snowplowanalytics.snowplow.storage.bigquery.generated.ProjectMetadata
+
+object ProcessorInfo {
+  val BQLoaderProcessorInfo = Processor(ProjectMetadata.name, ProjectMetadata.version)
```

Comment:
I'm wondering - we technically have at least two distinct processors: Loader and Repeater, but they both will have same processor info?

Reply:
Yeah, you are right. I will create them in the projects respectively.